projects/habitat_ovmm/configs/agent/heuristic_agent.yaml

```diff
@@ -5,6 +5,10 @@ radius: 0.05            # robot radius (in meters)
 fall_wait_steps: 200    # number of steps to wait after the object has been dropped
 store_all_categories: False  # whether to store all semantic categories in the map or just task-relevant ones
 detection_module: detic # the detector to use for perception in case ground_truth_semantics are turned off
```

Comment:
Thanks for this update! May consider moving detection_module inside `VISION` as well.

Reply:
@yvsriram I'm making hardware fixes in this branch https://github.com/facebookresearch/home-robot/pull/463/files 

we can make this change there as well

Reply:
#463